### PR TITLE
add the plotPsfZernTask in the pipelines

### PR DESCRIPTION
**Pull request by LR-inaf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,4 @@ pytest_session.txt
 
 # Existing Phosim Donut Templates
 policy/cwfs/donutTemplateData/phosimTemplates/
+.ruff.toml
```
Comment (jbkalmbach) [inline .gitignore]: Please don't add this here. We have a custom file for this repository to run with ts_precommit.

File: doc/.ipynb_checkpoints/versionHistory-checkpoint.rst [hunk 2]
```diff
@@ -0,0 +1,1935 @@
+.. py:currentmodule:: lsst.ts.wep
```
Comment (jbkalmbach) [inline doc/.ipynb_checkpoints/versionHistory-checkpoint.rst]: Remove this file please.

File: pipelines/_ingredients/.ipynb_checkpoints/donutVizGroupPipeline-checkpoint.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+description: donut viz pipeline tasks
```
Comment (jbkalmbach) [inline pipelines/_ingredients/.ipynb_checkpoints/donutVizGroupPipeline-checkpoint.yaml]: Please remove this checkpoint file.

File: pipelines/production/.ipynb_checkpoints/comCamRapidAnalysisDanishPipeline-checkpoint.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+description: rapid analysis pipeline for ComCam w/ danish zernike estimation
```
Comment (jbkalmbach) [inline pipelines/production/.ipynb_checkpoints/comCamRapidAnalysisDanishPipeline-checkpoint.yaml]: Please remove the checkpoint files.

File: pipelines/production/.ipynb_checkpoints/comCamRapidAnalysisPipeline-checkpoint.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+description: rapid analysis pipeline for ComCam
```
Comment (jbkalmbach) [inline pipelines/production/.ipynb_checkpoints/comCamRapidAnalysisPipeline-checkpoint.yaml]: Please remove the checkpoint file.

File: python/lsst/ts/wep/task/.ipynb_checkpoints/donutSourceSelectorTask-checkpoint.py [hunk 6]
```diff
@@ -0,0 +1,373 @@
+# This file is part of ts_wep.
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/task/.ipynb_checkpoints/donutSourceSelectorTask-checkpoint.py]: Please remove checkpoint file

File: python/lsst/ts/wep/task/.ipynb_checkpoints/pairTask-checkpoint.py [hunk 7]
```diff
@@ -0,0 +1,304 @@
+# This file is part of ts_wep.
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/task/.ipynb_checkpoints/pairTask-checkpoint.py]: Please remove checkpoint file

**Review by jbkalmbach — COMMENTED** [review]
Looks good. Please remove the checkpoint files and the addition in gitignore.

File: .gitignore (hunk 1 above)
Comment (LR-inaf) [reply .gitignore]: I have to remove these three lines?

Comment (teweitsai) [reply .gitignore]: The `.ruff.toml` was added by the `ts_precommit` in a certain version when we began to use the `toml`.
For example, https://github.com/lsst-ts/ts_m2gui/blob/develop/.gitignore#L147-L148

Comment (jbkalmbach) [reply .gitignore]: Just remove the `.ruff.toml`. `ts_wep` uses a slightly different one than the standard `ts_precommit` ruff file. We can work on updating `ts_wep` so we don't need that but that should be a separate issue.

**Review by jbkalmbach — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**